Objects/obmalloc.c

```diff
@@ -1517,11 +1517,13 @@ _PyMem_DebugRawRealloc(void *ctx, void *p, size_t nbytes)
 
     debug_alloc_api_t *api = (debug_alloc_api_t *)ctx;
     uint8_t *q;           /* base address of malloc'epad d block */
-    uint8_t *data;        /* p + 2*SST == pointer to data bytes */
+    uint8_t *r;
     uint8_t *tail;        /* data + nbytes == pointer to tail pad bytes */
     size_t total;         /* 2 * SST + nbytes + 2 * SST */
     size_t original_nbytes;
-    int i;
+    size_t serialno;
+#define ERASED_SIZE 16
```

Comment:
What do you think of increasing the size of the save buffer up to 512 bytes, so increase ERASED_SIZE to 256 bytes, to erase more bytes, and so catch more bugs?

At minimum, I propose to increa ERASED_SIZE to 64 bytes.

Reply:
I afraid 512 bytes will increase the stack usage too much. Memory allocation can be used very deep in the call stack. Why you want to save more bytes? If the very start or the very end of the memory block are corrupted, this likely will crash the program, I think this is enough.